content/blog/esc-versioning-launch/index.md

```diff
@@ -0,0 +1,102 @@
+2. Run `esc env version history <environment-name>` to view all revision history
+3. Run `esc env diff <environment-name>[@<version>] <environment-name>[@<version>]` to compare changes between versions
+4. Run `esc env version tag <environment-name>@<tag>` to assign a tag to the latest revision
+5.  Create a new environment using `esc env init <new-environment-name>` and import the previously edited environment using this syntax:
+```yaml
+imports:
+- <environment-name>@<tag>
+```
+6. Run `esc open <new-environment-name>` to open and confirm that you have imported the right environment version
+
+Check out Pulumi [ESC CLI documentation](/docs/esc-cli/) for more details on available options and commands.
+
+### Using the Console
+
+In your Pulumi Cloud Console,
+
+1. Open an environment, make a change, and ‘save’
+2. Go to the `Versions` Tab and select `Revision History` view to sell the history of changes made to the environment
+3. Click on a revision to see its details or see the differences between revisions using 'compare revisions'
+4. Add a tag to a revision by clicking on the Actions menu
+5. Go to ‘Tags’ view and copy the reference to the revision
+6. Create a new environment and import that environment revision using that tag
+
+<span style="color: red;">**Update Video**</span>
+{{< video title="Pulumi ESC Versioning Demo" src="" autoplay="true" loop="true" >}}
+
+## Real-world Scenarios
+Here are a few examples of how you might use these features:
+
+- **Global Infrastructure Update**: You need to deploy a new configuration to a global service spanning multiple regions. Tag the updated configuration as `Next` and deploy it to one region for testing. After verifying stability, you can confidently roll it out to other regions. If issues arise, quickly revert to the previous `Stable` version, ensuring resilient and minimally disruptive deployments.
+- **Platform Team Releases**: If you're a Platform team responsible for generating configurations, by tagging releases with labels like `v1.0-stable` and `v1.2-development`, you can clearly communicate which configurations are production-ready, keeping other teams informed about the ongoing development, and fostering seamless collaboration.
+- **Code Review and Debugging**: You're working in a development environment where the latest revision includes multiple recent changes, and you notice that the application is experiencing performance issues. By using the Compare Revisions Side-by-Side feature, you can quickly identify the specific changes that might be causing the problem, enabling faster troubleshooting and resolution.
+- **A/B Testing**: You can try A/B testing a new checkout flow (checkout-B) against your existing one (checkout-A). Using version tags, you can gradually roll out each to a subset of users, monitoring conversion rates. If checkout-B wins, seamlessly switch all users over with a simple environment reference update – no downtime, no disruption.
```

Comment:
I would leave this out, as while technically true for some really crude form of a/b testing, we are lacking a ton of features that a/b testing would have and is often done at a more granular level than environment.

Reply:
Removing A/B in the update